### PR TITLE
Bounce is back....

### PR DESCRIPTION
**Pull request by callumok2004**
image

### CODE REVIEW

File: websites/B/Bounce/dist/metadata.json [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
+    "itsbounce.net",
+    "itsbounce.com",
+    "itsbounce.uk",
+    "wearebounce.co.uk"
   ],
-  "version": "2.5.3",
-  "logo": "https://boun.cc/cdn/noshadow.png",
-  "thumbnail": "https://i.imgur.com/XmJTiZY.png",
+  "version": "3.0.0",
+  "logo": "https://www.itsbounce.net/branding/avatar.png",
```
Comment (ririxidev) [inline websites/B/Bounce/dist/metadata.json:20]: only imgur 😉

File: websites/B/Bounce/dist/metadata.json [hunk 2]
```diff
@@ -11,12 +11,14 @@
...
+  "thumbnail": "https://i.imgur.com/tlLbekj.png",
```
Comment (ririxidev) [inline websites/B/Bounce/dist/metadata.json:21]: image

File: websites/B/Bounce/presence.ts [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
+    largeImageKey: "logo",
+    details: settings.details,
+    state: settings.state,
+    smallImageText: `${data.presenter?.name || "AutoDJ"} is live!`,
+    buttons: [
+      {
+        label: "Visit website",
+        url: "https://itsbounce.net"
+      }
+    ]
```
Comment (ririxidev) [inline websites/B/Bounce/presence.ts:41]: image

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
fix deepscan error too 
image

File: websites/B/Bounce/dist/metadata.json (hunk 2 above)
Comment (callumok2004) [reply websites/B/Bounce/dist/metadata.json:21]: "preferably"

File: websites/B/Bounce/dist/metadata.json (hunk 1 above)
Comment (callumok2004) [reply websites/B/Bounce/dist/metadata.json:20]: we are allowed to use our own cdn, I know the rules xoxoxoxoxoxoxoxoxoxoxoxooxoxoxxoxoxoxoxoxoxoxoxoxoxoxooxoxox

Comment (Bas950) [reply websites/B/Bounce/dist/metadata.json:20]: image
Has been changed a while back

File: websites/R/Repeat/dist/metadata.json [hunk 4]
```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/I5G1Lvn.png",
   "thumbnail": "https://i.imgur.com/7RjRna8.png",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:17]: logo needs to be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Timeraa** [comment]
Is it gonna bounce into the trash can again?

**Pull request closed without merging**